src/runtime/Object.cpp

```diff
@@ -999,6 +999,8 @@ struct CompareIndexItem {
 template <typename ResultType, typename ResultBinder>
 static ResultType objectOwnPropertyKeys(ExecutionState& state, Object* self)
 {
+    // TODO turn-on gc if replace std::multiset to another gc-well-supported type
+    GC_disable();
```

Comment:
Why is `GC` disabled here?

Reply:
Because bdwgc may not cover std::multiset also

Reply:
I got it that `std::multiset` could be another issue